modules/mono/editor/Godot.NET.Sdk/Godot.NET.Sdk/Sdk/Sdk.props

```diff
@@ -74,8 +74,13 @@
 
   <!-- Godot DefineConstants. -->
   <PropertyGroup>
-    <!-- Define constant to identify Godot builds. -->
-    <GodotDefineConstants>GODOT</GodotDefineConstants>
+    <!-- Define constants to identify Godot builds and versions. -->
+    <GodotDefineConstants>
+      GODOT;
+      GODOT4;GODOT4_OR_GREATER;
+      GODOT4_1;GODOT4_1_OR_GREATER;GODOT4_0_OR_GREATER;
```

Comment:
Shouldn't we also add `GODOT4_0_4_OR_GREATER`?

Reply:
As I said in the discussion with akien, I wouldn't do that as it is easy enough to deal with `#if GODOT4_1_OR_GREATER || GODOT4_0_4_OR_GREATER` and it significantly increases the maintenance burden of this feature and makes it basically impossible to accurately generate the defines automatically.